### DISCUSSION TITLE
Plugins settings

### DISCUSSION THREAD

**Discussion by Tonaion02** [bevyengine/bevy#14735] [Q&A]
Is it possible to modify settings of a plugin after the creation of the application?

**Comment by JMS55** [comment]
No. Sometimes the plugins use the settings to insert resources into the app that _can_ be controlled at runtime though.

**Reply by Tonaion02** [reply]
Thx a lot for the answer!

**Comment by nuzzles** [comment]
It depends on the plugin. Most plugins will insert resources that can be modified after initialization.

Likewise, you can modify plugins in groups at insertion time via `PluginGroup.set(...)`

For example:
https://github.com/bevyengine/bevy/blob/a34651502c65ed48f4a13f422386d8903ebb584c/examples/tools/scene_viewer/main.rs#L30-L46

**Reply by Tonaion02** [reply]
Thx a lot for the answer!
